### PR TITLE
Ruelvls 7

### PR DESCRIPTION
**Pull request by DianaBernardo**
The endpoint is working and the tests are passing.

### CODE REVIEW

File: src/main/java/com/greenfox/lvlup/exception/ErrorMessage.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package com.greenfox.lvlup.model;
+package com.greenfox.lvlup.exception;
```
Comment (drexeverest) [inline src/main/java/com/greenfox/lvlup/exception/ErrorMessage.java]: DTOs could be under the model package in an own dto package

File: src/main/java/com/greenfox/lvlup/model/HolderDTO.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
+package com.greenfox.lvlup.model;
```
Comment (drexeverest) [inline src/main/java/com/greenfox/lvlup/model/HolderDTO.java:1]: DTOs could be in an own dto package

File: src/main/java/com/greenfox/lvlup/model/PitchDTO.java [hunk 3]
```diff
@@ -0,0 +1,123 @@
+package com.greenfox.lvlup.model;
```
Comment (drexeverest) [inline src/main/java/com/greenfox/lvlup/model/PitchDTO.java:1]: DTOs could be in an own dto package

File: src/main/java/com/greenfox/lvlup/model/PitchSetDTO.java [hunk 4]
```diff
@@ -0,0 +1,37 @@
+package com.greenfox.lvlup.model;
```
Comment (drexeverest) [inline src/main/java/com/greenfox/lvlup/model/PitchSetDTO.java:1]: DTOs could be in an own dto package

File: src/test/java/com/greenfox/lvlup/controller/PitchControllerTest.java [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+  PitchSetDTO pitchSetDTO = new PitchSetDTO();
+
+  @Test
+  public void getPitchesWithCorrectHeader() throws Exception {
+    mockMvc.perform(get("/pitches")
+        .header("userTokenAuth", token)
+        .contentType(MediaType.APPLICATION_JSON))
+        .andExpect(status().isOk())
+        //.andExpect(content().json(mockingElements.getSucessful()))
```
Comment (drexeverest) [inline src/test/java/com/greenfox/lvlup/controller/PitchControllerTest.java]: don't leave comments in your code

**Comment by adelbiro** [comment]
can you maybe add Süvi as a reviewer too, I'm not sure he can approve the request without being a reviewer?

File: src/test/java/com/greenfox/lvlup/controller/PitchControllerTest.java (hunk 5 above)
Comment (adelbiro) [inline src/test/java/com/greenfox/lvlup/controller/PitchControllerTest.java]: We shouldn't leave comments in the code, and also this commented out line is not needed anymore, so you could remove it  ? :)

File: src/main/java/com/greenfox/lvlup/model/MockingElements.java [hunk 6]
```diff
@@ -0,0 +1,47 @@
+package com.greenfox.lvlup.model;
+
+public class MockingElements {
```
Comment (adelbiro) [inline src/main/java/com/greenfox/lvlup/model/MockingElements.java]: do you use this class anywhere in your code? if not, we don't need in anymore

**Review by drexeverest — APPROVED** [review]

**Pull request merged**